tests/test_authorize.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+
+import falcon
+
+import pytest
+
+from falcon_mutualauth.mutualauth import Authorize
+
+
+class FakeRequest(object):
```

Comment:
It is not, it was just added to act like a request with which `Authorize` would interact.